### PR TITLE
Updated Connection/Connexion to Log in/Connected on main login page

### LINKED ISSUE #4806: Log in vs Connection button on initial login page
On the initial log in form, the button to log in is labeled "Connexion" (fr) or "Connection" (en). 
Shouldn't these be "Log in" (en) and "Connecter"/"Ouvrir la session" in French?

### PR DESCRIPTION
**Pull request by jmcameron**
Note: The French for "log in error" seemed correct, so I did not change it.

### CODE REVIEW

File: client/src/i18n/fr/form.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
             "CONFIGURE_PAYMENT": "Configurer pour le Paiement",
             "CONFIRM": "Confirmer",
             "CONFIRM_CANCEL": "Confirmer Annulation",
-            "CONNECTION": "Connexion",
+            "CONNECTION": "Connecter",
```
Comment (mbayopanda) [inline client/src/i18n/fr/form.json]: In good French, it is: `Se connecter`

**Review by mbayopanda — COMMENTED** [review]
There is a little comment about french translation

Use `Se connecter` instead of `Connecter`

File: client/src/i18n/fr/form.json (hunk 1 above)
Comment (jmcameron) [reply client/src/i18n/fr/form.json]: Thanks!  I'm fixing it.

Comment (jmcameron) [reply client/src/i18n/fr/form.json]: @mbayopanda I just pushed the fix.  Please verify  when you can and then tell Bors to release this tiny fix.

**Review by mbayopanda — APPROVED** [review]
bors r+

**Pull request merged**